common/src/PreferenceManager.h

```diff
@@ -197,6 +198,21 @@ namespace TrenchBroom {
         return prefs.get(preference);
     }
 
+    /**
+     * Sets a preference, and saves the change immediately.
+     */
+    template <typename T>
+    void setPref(Preference<T>& preference, const T& value) {
+        PreferenceManager& prefs = PreferenceManager::instance();
+        unused(prefs.set(preference, value));
```

Comment:
Why `unused` here?

Reply:
`prefs.set` returns a bool, I don't remember what it indicates off hand

Reply:
But you don't need to mark the expression as unused when you don't intend to use the result.

Reply:
I think it returns false if the new value is identical to the previous value, so you could even use the result to determine whether to call save at all. Not that it's terribly important, though.

Reply:
Oh OK, I was coding as if we have a warning for unused results. I guess it's not really practical in C++. Can drop this then.

Reply:
Only if the function or return type is marked as `[[nodiscard]]`.